crypto/x509/x509_ext.c

```diff
@@ -44,7 +44,14 @@ X509_EXTENSION *X509_CRL_get_ext(const X509_CRL *x, int loc)
 
 X509_EXTENSION *X509_CRL_delete_ext(X509_CRL *x, int loc)
 {
-    return X509v3_delete_ext(x->crl.extensions, loc);
+    X509_EXTENSIONS *ret = X509v3_delete_ext(x->crl.extensions, loc);
+    /* Empty extension lists are omitted. */
```

Comment:
nit: code style mandates a blank line above this to separate code and variable declarations.

Reply:
Haha, whoops, we crossed midair. Noticed these in CI after sending out the PR. Fixed.

Reply:
Slow down I must :grinning: